src/styles/card-block.scss

```
@@ -0,0 +1,108 @@
+.main {
+  display: grid;
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  padding: 50px 0 72px;
+}
+
+.card {
+  border: 1px solid $borderColor;
+  border-radius: 5px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+
+    &-picture {
+      max-width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__title {
+    transition: color 300ms;
+    margin: 40px 0 0;
+    color: $textColor;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &:hover .card__title {
+    color: #34568b;
+  }
+
+  &__product-code {
+    margin: 4px 0 0;
```

```suggestion
    margin-top: 4px;
```
if you set it only to one side